db/migrate/20221014124812_create_contents.rb

```diff
@@ -0,0 +1,9 @@
+class CreateContents < ActiveRecord::Migration[6.1]
+  def change
+    create_table :contents do |t|
+      t.text :body
+      t.belongs_to :lesson, foreign_key: true
```

Comment:
Nit: Can we include null false constrains on both the body and lesson_id please?  We'll just have to update `.create_content` to `.build_content` in the importer to accomodate it.

This is going to be great for verifying lesson Github paths are correct, it will make it so much easier 🔥

Reply:
Done ✅